package.json

```diff
@@ -237,7 +237,7 @@
     "sass-embedded": "^1.66.1",
     "sass-lint": "^1.13.1",
     "sass-lint-auto-fix": "^0.21.2",
-    "sass-loader": "npm:@bsfishy/sass-loader@10.4.1-support-sass-embedded.3",
+    "sass-loader": "npm:@amoo-miki/sass-loader@10.4.1-node-sass-9.0.0-libsass-3.6.5-with-sass-embedded.rc1",
```

Comment:
Non-blocking: what does this change?

Reply:
I don't think it does anything. I was just syncing deps with OSD's.